pages/resultset.py

```diff
@@ -30,9 +32,12 @@ def open_next_unclassified_failure(self):
         self.selenium.find_element(*self._resultset_locator).send_keys("n")
 
     def open_logviewer(self):
-        WebDriverWait(self.selenium, self.timeout).until(lambda s: self.selenium.find_element(*self._result_status_locator).is_displayed())
+        WebDriverWait(self.selenium, self.timeout).until(lambda s: self.selenium.find_element(*self._job_details_actionbar_locator).is_displayed())
         self.selenium.find_element(*self._resultset_locator).send_keys("l")
 
+    def return_unclassified_failure_count(self):
+        return self.selenium.find_element(*self.unclassified_failure_count_locator).get_attribute("value")
```

Comment:
This element doesn't appear to have an attribute named `value` so I suspect this will return `None`. I think you probably want the text content, coerced into an integer.